app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentPresenter.kt

```diff
@@ -64,89 +66,132 @@ class TopicLessonsFragmentPresenter @Inject constructor(
       container,
       /* attachToRoot= */ false
     )
-    binding.let {
-      it.lifecycleOwner = fragment
+    binding.apply {
+      this.viewModel = topicLessonViewModel
+      this.lifecycleOwner = fragment
```

Comment:
Suggested making this alphabetical order.

Reply:
done